content/_code-samples/send-a-memo/js/send-a-memo.js

```diff
@@ -0,0 +1,58 @@
+if (typeof module !== "undefined") {
+    // Use var here because const/let are block-scoped to the if statement.
+    var xrpl = require('xrpl')
+  }
+  // Validate and send a Memo
+  // https://xrpl.org/transaction-common-fields.html#memos-field
+  
+  async function main() {
+    // Connect to a testnet node
+    console.log("Connecting to Testnet...")
+    const client = new xrpl.Client('wss://s.altnet.rippletest.net:51233')
+    await client.connect()
+  
+    // Get account credentials from the Testnet Faucet
+    console.log("Requesting addresses from the Testnet faucet...")
+    const { wallet, balance } = await client.fundWallet()
+    
+    console.log("Account: ", wallet.address)
+    console.log("   Seed: ", wallet.seed)
+
+    // Enter memo data to insert into a transaction
+    const MemoData = xrpl.convertStringToHex(string="TEXT 123 -=+");
```

Comment:
I'd recommend avoiding the word `TEXT` in the actual memo text because it may be confused with the keyword `Text` used below in the `MemoType` object. (Either because they think that you need it, or causing people to think that the below `Text` is also arbitrary)
```suggestion
    const MemoData = xrpl.convertStringToHex(string="Example Memo - 123 -=+");
```